issuer-list/src/main/java/com/adyen/checkout/issuerlist/internal/ui/model/IssuerListComponentParamsMapper.kt

```diff
@@ -39,7 +39,11 @@ class IssuerListComponentParamsMapper(
         )
         return IssuerListComponentParams(
             commonComponentParams = commonComponentParamsMapperData.commonComponentParams,
-            isSubmitButtonVisible = componentConfiguration?.isSubmitButtonVisible ?: true,
+            isSubmitButtonVisible = if (commonComponentParamsMapperData.commonComponentParams.isCreatedByDropIn) {
```

Comment:
Minor: you can declare `val commonComponentParams = commonComponentParamsMapperData.commonComponentParams` before to make this and line above shorter